raiden_contracts/deploy/contract_verifier.py

```diff
@@ -349,13 +353,13 @@ def _verify_service_registry_deployment(
     """ Check an onchain deployment of ServiceRegistry and constructor arguments """
     if to_checksum_address(service_registry.functions.token().call()) != token_address:
         raise RuntimeError("service_registry has a wrong token address")
+    if len(constructor_arguments) != 2:
+        raise RuntimeError(
+            "service_registry was deployed with a wrong number of constructor arguments"
```

Comment:
Above you use CamelCase for the contract name

Reply:
I'll fix this together with similar occurrences above and below.